HACKING

```diff
@@ -766,9 +766,16 @@ In ASCII or UTF-32 mode, the character counts in OP_REVERSE and OP_VREVERSE are
 also the number of code units, but in UTF-8/16 mode each character may occupy
 more than one code unit.
 
-The "scan substring" assertion compiles as OP_ASSERT_SCS. What follows takes
-the same form as a conditional subpattern with a back reference condition (see
-next section).
+The "scan substring" assertion compiles as OP_ASSERT_SCS. This opcode is
+followed by a list of arguments (see next section). The first argument
+is always preceded by a META_OFFSET opcode, which sets the starting
```

Comment:
Can these two paragraphs possibly be right? You are saying that OP_ASSERT_SCS is followed by META_OFFSET?

META_SCS is followed by META_OFFSET; and OP_ASSERT_SCS is followed by OP_(DN)CREF, surely?